### PR TITLE
Adding styles sass directories into the git repo. 

### PR DESCRIPTION
**Pull request by abhi-g**
- Added the styles sass repo into the directory. 
     - Build process automatically updates style files in the theme/static dir for kf.
     - Simply run `gulp` from the sass src dir after updating sass files.
- Incorporated fixes to the style for header font sizes, Hero section height, whitespace around pre format/code  sections, left side content margin (can be centered further).
- Fixed templates for the mobile nav menu generation.

### CODE REVIEW

**Comment by ankushagarwal** [comment]
@abhi-g Can you update the development section of the top level README https://github.com/kubeflow/website#development and add instructions on how to update css using the sass files.

File: README.md [hunk 1]
```diff
@@ -14,3 +14,18 @@ macOS...sorry :-)
...
+### Making changes to CSS
+
+The css/sass style code is located in the `themes/kf/sass` directory. If you need
+to make changes to the style, you can update the themes style using:
+
+```
+cd themes/kf/sass
+# Note the npm install step is needed only the first time.
+npm install
+gulp
```
Comment (ankushagarwal) [inline README.md]: npm install doesn't seem to install gulp.

I had to install it separately using `npm install --global gulp-cli`

Can you document this?

Comment (abhi-g) [reply README.md]: done

**Pull request merged**